nimbus_verified_proxy/tests/data/download_proofs.sh

```
@@ -0,0 +1,132 @@
+curl --request POST \
```

It's convenient for shell scripts to have the shbang line, and also for things like these where it's generating data the correctness and completeness of which is important, using `set -Eeuo pipefail` will try to make explicit many of the failure modes bash/zsh-like shell scripts can have, rather than silently failing to run some of their commands.